tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -50,13 +50,16 @@ public async Task WamSilentAuthUserInteractionRequiredAsync()
             try
             {
                 var result = await pca.AcquireTokenSilent(scopes, PublicClientApplication.OperatingSystemAccount).ExecuteAsync().ConfigureAwait(false);
-
+                Assert.IsTrue(false, "AcquireTokenSilent successful!");
             }
             catch (MsalUiRequiredException ex)
             {
                 Assert.IsTrue(!string.IsNullOrEmpty(ex.ErrorCode));
             }
-
+            catch (Exception ex)
```

Comment:
nit: don't think this is needed. We can let the test just fail through.

Reply:
This is to catch any other exception